kernel/packages/unity-interface/UnityInterface.ts

```diff
@@ -188,6 +190,28 @@ export class UnityInterface {
     this.SendMessageToUnity('Main', 'SetDisableAssetBundles')
   }
 
+  public async CrashPayloadRequest(): Promise<string> {
+    // Over wasm this should come back on the same call stack frame because
+    // the response comes within the CrashPayloadRequest method body.
+
+    // For websocket this should take more frames, so we need promises.
+    let promise = new Promise<string>((resolve, reject) => {
+      let crashListener = this.crashPayloadResponseObservable.addOnce((payload) => {
+        resolve(payload)
+      })
```

Comment:
shouldn't be the listener removed from the observable also here?

Reply:
`addOnce` already handles the removal